xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd

```diff
@@ -1098,6 +1098,24 @@ Rail transport, Roads and Road transport
 				</xsd:annotation>
 			</xsd:element>
 			<xsd:element name="FullCharge" type="WattHoursType" minOccurs="0">
+			<xsd:element name="CustomerServiceContactDetails" type="ContactStructure" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Especially very large vehicles and also some ALTERNATIVE MODE OF OPERATION offers need the possibility to contact the VEHICLE directly.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+		</xsd:sequence>
+	</xsd:group>
+	<xsd:group name="VehicleCodeGroup">
+		<xsd:annotation>
+			<xsd:documentation>Alternative Code Elements for VEHICLE.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence>
+			<xsd:element name="OperationalNumber" type="xsd:normalizedString" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Operational Number of VEHICLE.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element ref="PrivateCode" minOccurs="0">
 				<xsd:annotation>
```

Comment:
The annotation below belongs to full charge.

Reply:
I know. Something is very wrong with this PR... Will have to check.